certbot/display/util.py

```diff
@@ -23,9 +23,6 @@
 CANCEL = "cancel"
 """Display exit code for a user canceling the display."""
 
-HELP = "help"
```

Comment:
I'm not sure if we want to delete this due to its usage [here](https://github.com/gboudreau/certbot-heroku/blob/2837a5bdfe527853311e57923557ec77b78c3317/certbot_heroku/configurator.py#L107). Did you have any thoughts about my comments [here](https://github.com/certbot/certbot/issues/3915#issuecomment-307519556) and did you check if `IDisplay`/`HELP` is used anywhere else?

If there aren't many plugins using it, I think we can open an issue/PR against their repos to notify them of the upcoming change. Alternatively, we can keep this around and update the docstring to state that it's deprecated.